zebra-state/src/constants.rs

```diff
@@ -13,3 +15,6 @@ pub const MAX_BLOCK_REORG_HEIGHT: u32 = MIN_TRANSPARENT_COINBASE_MATURITY - 1;
 
 /// The database format version, incremented each time the database format changes.
 pub const DATABASE_FORMAT_VERSION: u32 = 4;
+
+/// Portion of OS error when the RocksDB lock file is already open.
+pub const LOCK_FILE_ERROR: &str = "lock file";
```

Comment:
We need to do exactly the same check in the tests and the URL filter:
```suggestion
pub const LOCK_FILE_ERROR: &str = "(lock file).*(temporarily unavailable)|(in use)";
```